jplag/pom.xml

```diff
@@ -1,104 +1,109 @@
-						<manifest>
-							<mainClass>de.jplag.CLI</mainClass>
-							<packageName>jplag</packageName>
-						</manifest>
-					</archive>
-				</configuration>
-			</plugin>
-		</plugins>
-	</build>
-
-	<dependencies>
-		<dependency>
-			<groupId>net.sourceforge.argparse4j</groupId>
-			<artifactId>argparse4j</artifactId>
-		</dependency>
-		<dependency>
-			<groupId>com.github.stefanbirkner</groupId>
-			<artifactId>system-rules</artifactId>
-			<scope>test</scope>
-		</dependency>
-
-		<dependency>
-			<groupId>de.jplag</groupId>
-			<artifactId>frontend-utils</artifactId>
-		</dependency>
-		<dependency>
-			<groupId>de.jplag</groupId>
-			<artifactId>text</artifactId>
-		</dependency>
-		<dependency>
-			<groupId>de.jplag</groupId>
-			<artifactId>java</artifactId>
-		</dependency>
-		<dependency>
-			<groupId>de.jplag</groupId>
-			<artifactId>python-3</artifactId>
-		</dependency>
-		<dependency>
-			<groupId>de.jplag</groupId>
-			<artifactId>csharp-1.2</artifactId>
-		</dependency>
-		<dependency>
-			<groupId>de.jplag</groupId>
-			<artifactId>cpp</artifactId>
-		</dependency>
-		<dependency>
-			<groupId>de.jplag</groupId>
-			<artifactId>scheme</artifactId>
-		</dependency>
-		<dependency>
-			<groupId>de.jplag</groupId>
-			<artifactId>chars</artifactId>
-		</dependency>
-	</dependencies>
-</project>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+
+	<modelVersion>4.0.0</modelVersion>
+	<artifactId>jplag</artifactId>
+
+	<parent>
+		<groupId>de.jplag</groupId>
+		<artifactId>aggregator</artifactId>
+		<version>3.1.0-SNAPSHOT</version>
+	</parent>
+
+	<build>
+		<resources>
+			<resource>
+				<directory>src/main/resources</directory>
+				<filtering>true</filtering>
+			</resource>
+
+			<resource>
+				<directory>src/main/resources</directory>
+				<filtering>false</filtering>
+				<excludes>
+					<exclude>de/jplag/version.properties</exclude>
+				</excludes>
+			</resource>
+		</resources>
+
+		<plugins>
+			<plugin>
+				<artifactId>maven-jar-plugin</artifactId>
+				<configuration>
+					<archive>
+						<manifest>
+							<mainClass>de.jplag.CLI</mainClass>
+							<packageName>jplag</packageName>
+						</manifest>
+					</archive>
+				</configuration>
+			</plugin>
+
+			<plugin>
+				<groupId>org.apache.maven.plugins</groupId>
+				<artifactId>maven-shade-plugin</artifactId>
+				<configuration>
+					<transformers>
+						<transformer implementation="org.apache.maven.plugins.shade.resource.ServicesResourceTransformer"/>
+					</transformers>
+					<shadedArtifactAttached>true</shadedArtifactAttached>
+					<shadedClassifierName>jar-with-dependencies</shadedClassifierName>
+				</configuration>
+				<executions>
+					<execution>
+						<phase>package</phase>
+						<goals>
+							<goal>shade</goal>
+						</goals>
+					</execution>
+				</executions>
```

Comment:
The change is here.

It seems that the file was using CRLF lines instead of LF only